examples/cluster/plot_bisect_kmeans.py

```diff
@@ -6,9 +6,13 @@
 This example shows differences between Regular K-Means algorithm and Bisecting K-Means.
 
 While K-Means clusterings are different when with increasing n_clusters,
-Bisecting K-Means clustering build on top of the previous ones.
+Bisecting K-Means clustering build on top of the previous ones. As a
+result, it tends to create clusters that have a more consistent
+large-scale structure across the clusters.
```

Comment:
```suggestion
Bisecting K-Means clustering builds on top of the previous ones. As a
result, it tends to create clusters that have a more consistent
large-scale structure across the clusters.
```

k-means is plural but I think we treat is a a proper name for the estimator (singular). Furthermore it's more consistent with the use of "it" singular in the next sentence.

Reply:
And actually "clustering" is singular here.

Reply:
Right, you are correct and your suggestion is the proper phrasing. Will commit your change.